client/flutter/lib/main.dart

```diff
@@ -27,16 +39,17 @@ class _MyAppState extends State<MyApp> {
       localizationsDelegates: [
         GlobalMaterialLocalizations.delegate,
         GlobalWidgetsLocalizations.delegate,
-        S.delegate
+        S.delegate,
       ],
-      supportedLocales: S.delegate.supportedLocales,
+      supportedLocales: Platform.isMacOS ? [Locale('en', '')] : S.delegate.supportedLocales,
```

Comment:
We shouldn't have platform specific code in our main code base if we can avoid it. In this case localization is a known issue and we should fix that instead of adding a workaround for 1 platform.